internal/controller/nodeagent.go

```diff
@@ -270,20 +287,42 @@ func (r *DataProtectionApplicationReconciler) customizeNodeAgentDaemonset(ds *ap
 		},
 	)
 
+	privileged := true
+
+	if dpa.Spec.Configuration.Velero.DisableFsBackup != nil {
```

Comment:
since you are using default marker, is not this option always set?

Reply:
possibly, but better check then be sorry :)